src/sentry/api/endpoints/project_rules.py

```diff
@@ -800,6 +800,7 @@ def post(self, request: Request, project) -> Response:
             "name": data["name"],
             "environment": data.get("environment"),
             "project": project,
+            "project_id": project.id,
```

Comment:
Because `find_channel_id_for_rule` accepts `kwargs` adding this parameter will be safe. I'll still need a second change to remove the `project` parameter.